src/main/java/ch/naviqore/gtfs/schedule/model/Stop.java

```diff
@@ -18,7 +18,8 @@ public final class Stop implements Initializable, Location<GeoCoordinate> {
 
     private final String id;
     private final String name;
-    @Nullable private final String parentId;
+    @Nullable
+    private final String parentId;
```

Comment:
If we change this to Stop instead of StopId, then we should set the lombok Getter to None:

`@Getter(AccessLevel.None)` to prevent lombok from generating a getter and add one manually that returns Optional<Stop>, so we avoid the issue of NullPointers.

```java
Optional<Stop> getParent() {
    return Optional.ofNullable(parent)
}
```